tasks/phylogenetic_inference/task_ksnp3_shared_snps.wdl

```diff
@@ -0,0 +1,71 @@
+version 1.0
+
+task ksnp3_shared_snps {
```

Comment:
do we want to change this to be shared_variants instead of shared_snps to match the workflow name?